src/DayColumn.js

```diff
@@ -349,7 +349,7 @@ class DayColumn extends React.Component {
 
     while (dates.lte(current, endDate)) {
       slots.push(current)
-      current = dates.add(current, this.props.step, 'minutes')
+      current = new Date(+current + this.props.step * 60 * 1000)
```

Comment:
i think this is fine but lets add a comment explaining why here in the event this is addressed upstream